examples/using-typescript/README.md

```diff
@@ -0,0 +1,44 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
Change the readme to align with the other examples: https://github.com/RisingStack/react-easy-state/edit/master/examples/clock/README.md (Name + live link + short description)